specs/lib/gltfPipelineSpec.js

```diff
@@ -120,7 +120,7 @@ describe('gltfPipeline', function() {
         readGltf(gltfEmbeddedPath, options, function (gltf) {
             var initialUri = gltf.buffers.CesiumTexturedBoxTest.uri;
             processJSON(gltf, options, function () {
-                var finalUri = gltf.buffers.CesiumTexturedBoxTest.uri;
+                var finalUri = gltf.buffers.uri;
```

Comment:
Why did you make this change? If you do this you're comparing at different levels of the glTF hierarchy.